### ISSUE TITLE
Capturing Flags & Intel on Flags

### ISSUE THREAD

**Issue by PR3SIDENT** [olumby/unknown#5]
One thing I like about King of the Hill is being able to know how many enemies are also trying to capture flags. They do this really well with an interface at the bottom, which shows all the time. 

In wasteland... that information needs to be relative - otherwise the flags will become the focus and the game will just turn into a huge King of the Hill version. 

I think the way we can make flag capturing a little more intelligent is to show # of people IN a flag area to people also in that area. So for example, I wouldn't be able to see how many enemies are capturing a zone (or attempting to) unless I myself was in that zone. When I entered the zone I'd be presented with an interface that said "5 OPFOR, 4 BLUFOR, 6 IND" etc... Until I left the area in which I would no longer have that flag intel.

**Comment by olumby** [comment]
To be honest I had no indention of showing players how many players are in the zone, inner or outer. I wanted to follow more of a Capture the Flag approach only showing the player whether a flag was captured (and by who) or whether a flag is being captured (but not by who).

In king of the hill, the zone is much much larger than what I have currently set up. Even the outer zone is pretty small. If an opposing player managed to sneak into a flag undetected I think it would be slightly unfair to point them out. For example if 5 BLUFOR players are camping the inner zone, as soon as one OPFOR player enters it wouldn't take long for those BLUFORs to find and kill the OPFOR, even less so if they had friendlies watching from the outside, or a UAV. It also makes it much harder for a well coordinated opposing team to capture a flag as the holders would have a checklist as they take out enemies.

In conclusion, I don't think this would work well for smaller flags, but we can see how the current model plays out.
